MicrosoftSentinel/connector_microsoft_sentinel.json

```diff
@@ -0,0 +1,51 @@
+{
+    "description": "Forward Microsoft Sentinel Alerts to Sekoia",
+    "docker_parameters": "get_microsoft_sentinel_alerts",
+    "name": "Get Microsoft Sentinel alerts",
+    "arguments": {
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "properties": {
+        "frequency": {
+          "type": "integer",
+          "description": "Batch frequency in seconds",
+          "default": 60
+        },
+        "chunk_size": {
+          "type": "integer",
+          "description": "The max size of chunks for the batch processing",
+          "default": 1000
+        },
+        "intake_server": {
+          "description": "Server of the intake server (e.g. 'https://intake.sekoia.io')",
+          "default": "https://intake.sekoia.io",
+          "type": "string"
+        },
+        "intake_key": {
+          "description": "Intake key to use when sending events",
+          "type": "string"
+        },
+        "subscription_id": {
+          "description": "Subscription ID of the Microsoft Sentinel instance",
+          "type": "string"
+        },
+        "resource_group": {
+          "description": "Resource group of the Microsoft Sentinel instance",
+          "type": "string"
+        },
+        "workspace_name": {
+          "description": "Workspace name of the Microsoft Sentinel instance",
+          "type": "string"
+        }
+      },
```

Comment:
Can you move these fields into the module configuration?
Sorry for this change.